lib_xua/src/core/audiohub/xua_audiohub.xc

```diff
@@ -616,7 +617,11 @@ static void dummy_deliver(chanend ?c_out, unsigned &command)
                     }
 #endif
                 }
-
+                const int wait_ticks = XS1_TIMER_HZ/48000;
```

Comment:
Only issue with adding a timer here is it will slow down the DFU commands a bit - did you notice that at all?

Reply:
Hmm, I didn't notice the DFU slowing down.

Reply:
okay, lets go with it for now.